server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/ProjectTester.java

```diff
@@ -62,6 +62,6 @@ void deleteAll() {
       .setOrganization(organization != null ? organization.getKey() : null);
     stream(populators).forEach(p -> p.accept(request));
 
-    return session.wsClient().projects().create(request.build()).getProject();
+    return session.wsClient().projectsOld().create(request.build()).getProject();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "projectsOld"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)